node/pkg/logscribe/app.go

```diff
@@ -57,8 +106,35 @@ func Run(ctx context.Context) error {
 	return nil
 }
 
-func bulkCopyLogs(ctx context.Context, logsChannel <-chan *[]api.LogInsertModel) {
-	ticker := time.NewTicker(DefaultBulkLogsCopyInterval)
+func (a *App) startProcessingCronJob(ctx context.Context) error {
+	services, err := db.QueryRows[Service](ctx, getServicesQuery, nil)
+	if err != nil {
+		log.Error().Err(err).Msg("Failed to get services")
+		return err
+	}
+
+	if a.cron == nil {
+		cron := cron.New()
+		_, err := cron.AddFunc("@weekly", func() { // Run once a week, midnight between Sat/Sun
```

Comment:
can you make it run every Thursday night, so that we can get fresh issues on maintenance day?